cli/oauth.go

```diff
@@ -1 +1,120 @@
 package cli
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/coreos/issue-sync/cfg"
+	"github.com/dghubble/oauth1"
+)
+
+// getJIRAHTTPClient obtains an access token (either from configuration
+// or from an OAuth handshake) and creates an HTTP client that uses the
+// token, which can be used to configure a JIRA client.
+func getJIRAHTTPClient(config cfg.Config) (*http.Client, error) {
```

Comment:
This creates a new Jira HTTP client, right? Let's call the the func `newJiraHTTPClient`. In general, Go does _not_ use getters. Typically, either a struct is accessible as a field or we create a new struct.